pkg/extensions/tbtc/tbtc.go

```diff
@@ -24,9 +26,24 @@ import (
 var logger = log.Logger("tbtc-extension")
 
 const (
-	maxActAttempts            = 3
-	pastEventsLookbackBlocks  = 10000
+	// Maximum number of action attempts before giving up and returning
+	// a monitoring error.
+	maxActAttempts = 3
+
+	// Determines how many blocks from the past should be included
+	// during the past events lookup.
+	pastEventsLookbackBlocks = 10000
+
+	// Number of blocks which should elapse before confirming
+	// the given chain state expectations.
 	defaultBlockConfirmations = 12
+
+	// Determines how long the monitoring cache will maintain
+	// its entries.
+	monitoringCachePeriod = 24 * time.Hour
+
+	// Used to calculate the action delay factor for the given signer index.
```

Comment:
`... to avoid all signers executing the same action for deposit at the same time` ?